src/controller/cve-id.controller/cve-id.controller.js

```diff
@@ -5,6 +5,8 @@ const CONSTANTS = require('../../constants')
 const utils = require('../../utils/utils')
 const errors = require('./error')
 const error = new errors.CveIdControllerError()
+const options = CONSTANTS.PAGINATOR_OPTIONS
+options.sort = { owning_cna: 'asc', cve_id: 'asc' }
```

Comment:
Question: why are we adding a `sort` option here, instead of from `src/constants/index.js`?

Reply:
The sorting option depends on the items that are being sorted. Here, the result is a list of cve-ids that are first sorted by the `owning_cna` and then by `cve-id` fields. In the Org controller, the sorting must be based on an org field such as the `short_name`.